configs/alpine.sh

```diff
@@ -28,7 +28,7 @@ extra_packages="$extra_apk_packages"
 # Opam switch to use. This determines the OCaml version and a set of
 # configuration options.
 
-opam_switch="4.10.0+flambda"
+opam_switch="4.10.2+flambda"
```

Comment:
I am a bit curious about why we use the _flambda_ compiler here, do we get any important optimization from it?

Reply:
@mjambon added that I think